cmd/ims/main.go

```diff
@@ -1,7 +1,46 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/sirupsen/logrus"
+	flag "github.com/spf13/pflag"
+
+	"github.com/omc-college/management-system/pkg/ims/api/routers"
+	"github.com/omc-college/management-system/pkg/ims/service"
+	"github.com/omc-college/management-system/pkg/config"
+)
 
 func main() {
-	fmt.Println("I'm identity management service")
+	var conf Config
+	var err error
+
+	configPath := flag.StringP("config", "c", "./ims-service-example-config.yaml", "path to service config")
+
+	flag.Parse()
+
+	err = config.Load(&conf, *configPath)
+	if err != nil {
+		logrus.Fatalf("cannot load config: %s", err.Error())
+	}
+
+	dsn := fmt.Sprintf("user=%s password=%s host=%s port=%s database=%s sslmode=%s",
+		conf.DBConnection.User, conf.DBConnection.Password, conf.DBConnection.Host,
+		conf.DBConnection.Port, conf.DBConnection.Database, conf.DBConnection.Sslmode)
+
+	// Open DB
+	db, err := sqlx.Connect("pgx", dsn)
+	if err != nil {
+		logrus.Fatalf("cannot connect to DB: %s", err.Error())
+	}
+
+	defer db.Close()
+
+	signupService := service.NewIMSService(db, nil, time.Time{})
```

Comment:
what means that arg? `time.Time{}`

Reply:
It`s a nil value of type time.Time

Reply:
I can`t just write `nil` here, it occurs an error, so I use a nil value of type time.Time

Reply:
please make config property for it